code/modules/mob/new_player/sprite_accessories_taur.dm

```diff
@@ -356,3 +363,72 @@
 /datum/sprite_accessory/tail/taur/zorgoia/fat
 	name = "Zorgoia (Fat Taur)"
 	extra_overlay = "zorgoia_fat"
+<<<<<<< HEAD
+=======
+
+
+/datum/sprite_accessory/tail/taur/wolf
+	vore_tail_sprite_variant = "N"
+	fullness_icons = 3
+	struggle_anim = TRUE
+
+/datum/sprite_accessory/tail/taur/naga/naga_2c
+	vore_tail_sprite_variant = "Naga"
+	fullness_icons = 1
+	struggle_anim = TRUE
+
+/datum/sprite_accessory/tail/taur/horse
+	vore_tail_sprite_variant = "Horse"
+	fullness_icons = 1
+	struggle_anim = TRUE
+
+/datum/sprite_accessory/tail/taur/cow
+	vore_tail_sprite_variant = "Cow"
+	fullness_icons = 1
+	struggle_anim = TRUE
+
+/datum/sprite_accessory/tail/taur/lizard
+	vore_tail_sprite_variant = "Lizard"
+	fullness_icons = 1
+	struggle_anim = TRUE
+
+/datum/sprite_accessory/tail/taur/lizard/synthlizard
+	vore_tail_sprite_variant = "SynthLiz"
+	fullness_icons = 1
+	struggle_anim = TRUE
+
+/datum/sprite_accessory/tail/taur/feline
+	vore_tail_sprite_variant = "Feline"
+	belly_variant_when_loaf = TRUE
+	fullness_icons = 1
+	struggle_anim = TRUE
+
+/datum/sprite_accessory/tail/taur/slug
+	vore_tail_sprite_variant = "Slug"
+	fullness_icons = 1
+	struggle_anim = TRUE
+
+/datum/sprite_accessory/tail/taur/drake
+	vore_tail_sprite_variant = "Drake"
+	belly_variant_when_loaf = TRUE
+	fullness_icons = 1
+	struggle_anim = TRUE
+
+/datum/sprite_accessory/tail/taur/otie
+	vore_tail_sprite_variant = "Otie"
+	belly_variant_when_loaf = TRUE
+	fullness_icons = 1
+	struggle_anim = TRUE
+
+/datum/sprite_accessory/tail/taur/deer
+	vore_tail_sprite_variant = "Deer"
+	belly_variant_when_loaf = TRUE
+	fullness_icons = 1
+	struggle_anim = TRUE
+
+/datum/sprite_accessory/tail/taur/skunk
+	vore_tail_sprite_variant = "Skunk"
+	belly_variant_when_loaf = TRUE
+	fullness_icons = 1
+	struggle_anim = TRUE
+>>>>>>> 8ff0932889... Adds Toggleable Stomachs w/ toggleable visibility (#16052)
```

Comment:
likely also copied from the modular_chomp, same as comment above

Reply:
A good chunk of these don't even have struggle anim sprites in the dmi despite being set to TRUE here. Gonna have some invisible guts. Only "N" "E" and "SectDrone" versions have anims

Reply:
Same with the _vr file below.